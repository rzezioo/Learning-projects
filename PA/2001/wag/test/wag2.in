3
100 10
2 7
1 4
