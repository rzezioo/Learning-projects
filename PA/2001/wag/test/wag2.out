2 7
1 4
100 10
